src/promptflow/promptflow/_core/tools_manager.py

```diff
@@ -163,6 +163,33 @@ def _load_package_tool(tool_name, module_name, class_name, method_name, node_inp
         # Return the init_inputs to update node inputs in the afterward steps
         return getattr(provider_class(**init_inputs_values), method_name), init_inputs
 
+    @staticmethod
+    def _load_script_tool(module, tool_name, class_name, method_name, node_inputs: Mapping[str, InputAssignment]):
+        """Load script in tool with given import path and node inputs."""
+        if class_name is None:
+            return getattr(module, method_name), {}
+        provider_class = getattr(module, class_name)
+        # Note: v -- type is InputAssignment
+        init_inputs = provider_class.get_initialize_inputs()
```

Comment:
Seems similar to package tool loading? Can we share some code so we could have more clear UT?